lib/lib-openapi/src/test/resources/controller/ComplexExampleController.java

```diff
@@ -473,4 +475,16 @@ public void getUserId(
 			@RequestParam(value = "userId") Long userId
 	) {
 	}
+
+	/**
+	 * Example of an endpoint that requires authorization
+	 *
+	 * @param userId
+	 */
+	@RequiredScope({OAuthScope.view, OAuthScope.modify})
+	@RequestMapping(value = "/complex-pet/authorization", method = RequestMethod.GET)
+	public void getAuthorization(
+			@RequestParam(value = "userId") Long userId
+	) {
+	}
```

Comment:
we need two cases: one that includes the userId (requires auth), and one missing userId, no auth needed.